doc/pages/Manual.md

```diff
@@ -1175,6 +1175,7 @@ files.
     - `OPAM_PACKAGE_VERSION=<ver>` (`<ver>` is the version of the package being built/installed/removed)
     - `OPAMCLI=2.0` (since opam 2.1)
     - `TMP` and `TMPDIR` are set by the sandbox script (bubblewrap), but should not be relied on since the sandbox is not used on all platforms and can be disabled by the user.
+    - `CYGWIN=winsymlinks:native` (or `CYGWIN=$CYGWIN winsymlinks:native` if CYGWIN is defined and not empty and CYGWIN does not contain either `winsymlinks:native` or `winsymlinks:nativestrict` already) on Wind    ows (since opam 2.2)
```

Comment:
```suggestion
    - `CYGWIN=winsymlinks:native` (or `CYGWIN=$CYGWIN winsymlinks:native` if `CYGWIN` is defined and not empty and `CYGWIN` does not contain either `winsymlinks:native` or `winsymlinks:nativestrict` already) on Windows (since opam 2.2)
```